packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1918,6 +1938,14 @@ class ImportedSubnet extends Resource implements ISubnet, IPublicSubnet, IPrivat
     return this._availabilityZone;
   }
 
+  public get ipv4CidrBlock(): string {
+    if (!this._ipv4CidrBlock) {
+      // tslint:disable-next-line: max-line-length
+      throw new Error("You cannot reference an imported Subnet's IPv4 CIDR if it was not supplied. Add the ipv4CidrBlock when importing using Subnet.fromSubnetAttributes()");
```

Comment:
I think the linter might yell at you for using `"` instead of `'` here.

Reply:
It didn't yell about using `"` instead of `'` and it looks like that's already in the source code for another function:

https://github.com/aws/aws-cdk/blob/8c2d7fca11bb48f43f37dc42e25de47147795452/packages/%40aws-cdk/aws-ec2/lib/vpc.ts#L1913-L1919

Will fix both